### PR TITLE
move swift examples

### PR DESCRIPTION
**Pull request by ubbabeck**
Added swift examples and workflow.

### CODE REVIEW

File: snippets/swift/BreezSDKExamples/Sources/ReceivePayment.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+func receivePayment(sdk: BlockingBreezServices) -> ReceivePaymentResponse? {
+    // ANCHOR: receive-payment
+    do {
+      let repsonse = try sdk.receivePayment(
+        req: ReceivePaymentRequest(
+            amountMsat: 3_000_000,
+            description: "Invoice for 3 000 000 sats"))
+        return repsonse
+    // ANCHOR_END: receive-payment
```
Comment (ok300) [inline snippets/swift/BreezSDKExamples/Sources/ReceivePayment.swift:18]: This should be after the `catch`

File: snippets/swift/BreezSDKExamples/Sources/SendPayment.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import BreezSDK
+
+func sendPayment(sdk: BlockingBreezServices) -> SendPaymentResponse? {
+    // ANCHOR: send-payment
+    do {
+        // The `amountMsat` param is optional and should only passed if the bolt11 doesn't specify an amount.
+        // The amountMsat is required in case an amount is not specified in the bolt11 invoice'.
+        let req = SendPaymentRequest(bolt11: "...", amountMsat: 3_000_000)
+        let response = try sdk.sendPayment(req: req)
+        // ANCHOR_END: send-payment
```
Comment (ok300) [inline snippets/swift/BreezSDKExamples/Sources/SendPayment.swift:17]: This should also be after the `catch`

File: snippets/swift/BreezSDKExamples/Sources/SendSpontaneous.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+func sendSpontaneousPayment(sdk: BlockingBreezServices) -> SendPaymentResponse? {
+    // ANCHOR: send-spontaneous-payment
+    let nodeId = "...";
+    do {
+        let response = try sdk.sendSpontaneousPayment(
+            req: SendSpontaneousPaymentRequest(
+                nodeId: "...",
+                amountMsat: 3_000_000))
+        // ANCHOR_END: send-spontaneous-payment
```
Comment (ok300) [inline snippets/swift/BreezSDKExamples/Sources/SendSpontaneous.swift:20]: Same as above

File: snippets/swift/BreezSDKExamples/Sources/ListPayments.swift [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+//
+//  Created by ruben on 13/11/2023.
+//
+
+import Foundation
+import BreezSDK
+
+func ListPayments(sdk: BlockingBreezServices) -> [Payment]? {
+    // ANCHOR: list-payments
+    let payments = try? sdk.listPayments(req: ListPaymentsRequest(filters: [.closedChannel, .sent, .received]))
```
Comment (ok300) [inline snippets/swift/BreezSDKExamples/Sources/ListPayments.swift:13]: In the other language snippets, `filters` is empty or not even set.

**Review by ok300 — CHANGES_REQUESTED** [review]
A few findings so far:
- the Supporting Fiat Currencies page has no swift snippets
- in some try / catch cases, the snippet abruptly ends at the end of the try, instead of including the catch (see above)

File: snippets/swift/BreezSDKExamples/Sources/ReceivePayment.swift (hunk 1 above)
Comment (ubbabeck) [reply snippets/swift/BreezSDKExamples/Sources/ReceivePayment.swift:16]: Fixed

File: snippets/swift/BreezSDKExamples/Sources/SendSpontaneous.swift (hunk 3 above)
Comment (ubbabeck) [reply snippets/swift/BreezSDKExamples/Sources/SendSpontaneous.swift]: Fixed

File: snippets/swift/BreezSDKExamples/Sources/SendPayment.swift (hunk 2 above)
Comment (ubbabeck) [reply snippets/swift/BreezSDKExamples/Sources/SendPayment.swift]: fixed

File: snippets/swift/BreezSDKExamples/Sources/ListPayments.swift (hunk 4 above)
Comment (ubbabeck) [reply snippets/swift/BreezSDKExamples/Sources/ListPayments.swift]: fixed

**Pull request merged**